src/test/java/org/folio/edge/patron/MainVerticleTest.java

```diff
@@ -124,33 +127,39 @@ public void testAdminHealth(TestContext context) {
   public void testGetAccountUnknownApiKey(TestContext context) throws Exception {
     logger.info("=== Test getAccount with unknown apiKey (tenant) ===");
 
+    int expectedStatusCode = 401;
+
     final Response resp = RestAssured
       .get(String.format("/patron/account/%s?apikey=%s", extPatronId, unknownTenantApiKey))
       .then()
-      .statusCode(401)
-      .header(HttpHeaders.CONTENT_TYPE, TEXT_PLAIN)
+      .statusCode(expectedStatusCode)
+      .header(HttpHeaders.CONTENT_TYPE, APPLICATION_JSON)
       .extract()
       .response();
 
-    String actual = resp.body().asString();
-    assertEquals(MSG_ACCESS_DENIED, actual);
+    ErrorMessage msg = ErrorMessage.fromJson(resp.body().asString());
+    assertEquals(MSG_ACCESS_DENIED, msg.errorMessage);
+    assertEquals(expectedStatusCode, (long)msg.httpStatusCode);
```

Comment:
This cast (and all the ones below) seems odd: expectedStatusCode in an "int", msg.httpStatusCode is an Integer. Casting to a long will require a call to msg.httpStatusCode.intValue() (which could NPE if it can be null) to deal with autoboxing and then a cast to long as well as expectedStatusCode to be cast to a long. That being said, is there a reason msg.httpStatusCode cannot be an int? Then again, this is only a test. :) But, I think, still something to consider...

Reply:
Yes, there is a reason for it.  The compiler thought it was ambiguous between AssertEquals(Object, Object) and AssertEquals(long, long) with the ints being passed in. It doesn't have the option for AssertEquals(int, int). I had to cast one of the parameter to a long to make it happy.  That being said, I just tried to cast it to an int and it is happy with it so I'll do it to make it a little more efficient.

Reply:
Yeah, I knew there'd be an ambiguous problem here, which is why I was wondering if the msg.httpStatusCode could be a primitive int instead of an Integer. No big deal, since, as far as I can tell, we never use the httpStatusCode directly, expect for the tests. Another option is to use an Integer for the expected value like: Integer expectedStatusCode = Integer.valueOf(401);. At least with two objects it shouldn't be ambiguous.

Reply:
Good point. I didn't realize that I was using Integer instead of int.  Will make that change.